### PR TITLE
Update hosting-tiers-table.md

### PR DESCRIPTION
**Pull request by Pilargit12**
Reviews sent by Olena - corrected.

### CODE REVIEW

**Review by StevenBarre — APPROVED** [review]

File: src/docs/platform-architecture-reference/hosting-tiers-table.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ The aim is to provide a clear and easily understandable overview of each OCP edi
 | **Target up time (availability)** | Platform availability for the Silver hosting tier is 90% for single-node and 99.5% for multi-node deployments |
 | [**Maximum data sensitivity**](https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/information-security/information-security-classification) | - Protected B: Stored at rest (in a database or file storage)<br> - Protected C: Processing only (must be stored elsewhere) |
 | **Integrated HA, DR options**| TransportServerClaims|
-| **Underlying release plan** | Aim for N minus 1 tracking Red Hat point releases|                  
+| **Underlying release plan** | Aim for N minus 1 tracking Red Hat point releases for OpenShift|
```
Comment (mitovskaol) [inline src/docs/platform-architecture-reference/hosting-tiers-table.md:42]:
```suggestion
| **Openshift upgrade plan** | Aim for N minus 1 tracking Red Hat point releases for OpenShift|                  
```

File: src/docs/platform-architecture-reference/hosting-tiers-table.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ The aim is to provide a clear and easily understandable overview of each OCP edi
 | **Target up time (availability)** | Platform availability for the Emerald hosting tier is 90% for single-node application deployments and 99.5% for multi-node deployments |
 | [**Maximum data sensitivity**](https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/information-security/information-security-classification) | Protected C - Storage and/or processing |
 | **Integrated HA, DR options**| None|
-| **Underlying release plan** | - Extended Update Support (EUS) <br> - Only updates to even numbered releases|
+| **Underlying release plan** | - Extended Update Support (EUS) <br> - Only updates to even numbered releases of OpenShift|
```
Comment (mitovskaol) [inline src/docs/platform-architecture-reference/hosting-tiers-table.md:71]:
```suggestion
| **Openshift upgrade plan** | - Extended Update Support (EUS) <br> - Only upgrades to even numbered releases of OpenShift|
```

File: src/docs/platform-architecture-reference/hosting-tiers-table.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ The aim is to provide a clear and easily understandable overview of each OCP edi
...
 | **Slate of supported operators**| - Horizontal Pod Autoscaler<br> - Vertical Pod Autoscaler<br> - Custom Metrics Autoscaler<br> - Pipelines: Tekton<br> - CrunchyDB<br> - ArgoCD<br> - Kyverno <br>- License required Red Hat integration<br> - Enterprise DB <br> - JBoss  EAP <br> - IMB MQ |
```
Comment (mitovskaol) [inline src/docs/platform-architecture-reference/hosting-tiers-table.md:72]:
```suggestion
| **Supported operators**| - Horizontal Pod Autoscaler<br> - Vertical Pod Autoscaler<br> - Custom Metrics Autoscaler<br> - Pipelines: Tekton<br> - CrunchyDB<br> - ArgoCD<br> - Kyverno <br>- License required Red Hat integration<br> - Enterprise DB <br> - JBoss  EAP <br> - IMB MQ |
```

File: src/docs/platform-architecture-reference/hosting-tiers-table.md [hunk 4]
```diff
@@ -54,7 +54,7 @@ The aim is to provide a clear and easily understandable overview of each OCP edi
 | **Target up time (availability)** | Platform availability for the Gold hosting tier is 99.95%, specifically for applications with multi-node deployments that have a geographic failover to the Gold DR cluster in the Calgary data centre, minimizing disruption. <br><br>Without DR, Gold maintains the same availability as Silver |
 | [**Maximum data sensitivity**](https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/information-security/information-security-classification) | - Protected B: Stored at rest (in a database or file storage)<br> - Protected C: Processing only (must be stored elsewhere) |
 | **Integrated HA, DR options**| - TransportServerClaims to connect between Gold and Gold DR<br> -Global Load Balancing between clusters for Active/Passive DR|
-| **Underlying release plan** | Aim for N minus 1 tracking Red Hat point releases|
+| **Underlying release plan** | Aim for N minus 1 tracking Red Hat point releases for OpenShift|
```
Comment (mitovskaol) [inline src/docs/platform-architecture-reference/hosting-tiers-table.md:57]:
```suggestion
| **Openshift upgrade plan** | Aim for N minus 1 tracking Red Hat point releases for OpenShift|
```

**Review by mitovskaol — APPROVED** [review]

**Review by NickCorcoran — APPROVED** [review]
This content looks correct for Maximum data sensitivity.

**Pull request merged**